package.json

```diff
@@ -509,33 +631,24 @@
           "default": [],
           "description": "The Additional Config Arguments, .e.g [\"--cc=gcc\", \"--myopt=xxx\"]"
         },
-        "xmake.runningTargetsArguments": {
-          "type": "object",
-          "default": {
-            "default": []
-          },
-          "description": "The Running Targets Arguments, .e.g {\"targetName\": [\"args\", \"...\"]}"
-        },
-        "xmake.debuggingTargetsArguments": {
-          "type": "object",
-          "default": {
-            "default": []
-          },
-          "description": "The Debugging Targets Arguments, .e.g {\"targetName\": [\"args\", \"...\"]}"
```

Comment:
why? please revert it.

Reply:
It can configured with args in launch.json